### PR TITLE
Dbm 0430a

### PR DESCRIPTION
**Pull request by dbmcclain**
clean slate of system for start of generate-tree and reconstruct-tree. Can now generate-tree and then init-sim, and run successfully to completion.

### CODE REVIEW

File: src/Cosi-BLS/cosi-construction.lisp [hunk 1]
```diff
@@ -34,12 +34,14 @@ THE SOFTWARE.
 ;; --------------------------------------------------------------------
 ;; Physical network
 
-(defvar *local-nodes*  '(("Arroyo.local"    . "127.0.0.1" #|"10.0.1.33"|#)
+(defvar *local-nodes*  '(("Arroyo.local"    . "127.0.0.1")))
```
Comment (easye) [inline src/Cosi-BLS/cosi-construction.lisp]: @dbmcclain Let's change `Arroyo.local`  to `localhost` which would be meaningful to others than just your network.

File: src/Cosi-BLS/cosi-construction.lisp [hunk 2]
```diff
@@ -264,6 +266,16 @@ THE SOFTWARE.
           (coerce collected 'vector))
     ))
 
+(defun init-mappings ()
+  (setf *my-node* nil
+        *top-node* nil
+        *leader-node* (get-local-ipv4 "Arroyo.local")
```
Comment (easye) [inline src/Cosi-BLS/cosi-construction.lisp]: @dbmcclain Change to `localhost`?

**Review by easye — CHANGES_REQUESTED** [review]

**Comment by guitarvydas** [comment]
CL-USER 5 > (cosi-simgen::generate-tree :nodes 3)
#<COSI-SIMGEN::NODE 40200108AB>

CL-USER 6 > (cosi-simgen::init-sim)
65001

CL-USER 7 > (cosi-simgen::tst-blk)
#<ACTORS:ACTOR 402004280B>

CL-USER 8 > cosi-simgen::*leader*
NIL

"Construct Genesis transaction" 
"Find UTX for Mary" 
"Construct 2nd transaction" 
:SENT-TO-NULL 
(:MAKE-BLOCK)

**Comment by easye** [comment]
Good news: I figured out a way around the problems. They have nothing to do with setting specials, but rather if one calls cosi-simgen::shutdown-server without starting a server, there is no UDP traffic.

My simple fix on https://github.com/emotiq/emotiq/tree/dbm-0430a-mte is to put a simple numeric count around shutting down the server like https://github.com/emotiq/emotiq/commit/50cb1d1f70f63922879f45c44df7c8dd4ff7725f

**Review by easye — APPROVED** [review]

**Pull request merged**